README.md

```diff
@@ -175,6 +175,23 @@ You can redefine those variables in your `settings.py` :
   then it is in your `STATIC_ROOT` after you collected your
   [static files](https://docs.djangoproject.com/en/3.1/howto/static-files/).
 - `DJANGO_VITE_LEGACY_POLYFILLS_MOTIF` : The motif used to find the assets for polyfills inside the `manifest.json` (only if you use [@vitejs/plugin-legacy](https://github.com/vitejs/vite/tree/main/packages/plugin-legacy)).
+- `DJANGO_VITE_STATIC_URL_PREFIX` : prefix directory of your static files built by Vite.    
```

Comment:
Can you provide the default value for this variable in the doc.
Can you make sub-bullet points for each item "You ...", sorry I'm a little maniac 😊

Reply:
Done! It should be better now.
I totally get it! I'm a little maniac too :)

Reply:
😂